android/src/main/java/com/brentvatne/common/api/ControlsConfig.kt

```diff
@@ -21,6 +22,7 @@ class ControlsConfig {
                 config.hideDuration = ReactBridgeUtils.safeGetBool(src, "hideDuration", false)
                 config.hideNavigationBarOnFullScreenMode = ReactBridgeUtils.safeGetBool(src, "hideNavigationBarOnFullScreenMode", true)
                 config.hideNotificationBarOnFullScreenMode = ReactBridgeUtils.safeGetBool(src, "hideNotificationBarOnFullScreenMode", true)
+                config.liveLabel = ReactBridgeUtils.safeGetString(src, "liveLabel", null)
```

Comment:
Can we rename `src` to something else ? like `controlsConfig` or sm else

Reply:
Yeah, we can do it. Let me handle that.

Reply:
@KrzysztofMoch 
Done.